### PR TITLE
Refactor/related articles

### PR DESCRIPTION
**Pull request by colincclark**
- moves the related article container and heading/title to the article package in preparation for the slice package to be all about the templates

NOTE:

1. The related article stories will be grouped in another PR with input from the business as to how they want this to happen
2. The Slice pacakge tests here are temporary - they duplicate the related articles tests right now, but will eventually test the template code that will replace this

### CODE REVIEW

File: packages/article/article.stories.js [hunk 1]
```diff
@@ -22,6 +24,24 @@ const articleFixtureNoStandfirstNoFlags = require("./fixtures/no-standfirst-no-f
 const articleFixtureNoLabelNoFlags = require("./fixtures/no-label-no-flags.json");
 const articleFixtureNoLabelNoFlagsNoStandFirst = require("./fixtures/no-label-no-flags-no-standfirst.json");
 const articleFixtureNoLeadAsset = require("./fixtures/no-lead-asset.json");
+// Related articles
```
Comment (colincclark) [inline packages/article/article.stories.js]: the plan is to move all of the stories around on a separate PR

File: packages/article/__tests__/related-articles-shared.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import React from "react";
+import renderer from "react-test-renderer";
+import context from "jest-context";
+
+import RelatedArticles from "../related-articles/related-articles";
+
+const singleRelatedArticleFixture = require("../related-articles/fixtures/single-related-article.json");
+const singleRelatedArticleNoImageFixture = require("../related-articles/fixtures/single-related-article-no-image.json");
+const singleRelatedArticleNoLabelFixture = require("../related-articles/fixtures/single-related-article-no-label.json");
+const singleRelatedArticleNoBylineFixture = require("../related-articles/fixtures/single-related-article-no-byline.json");
```
Comment (nikhedonia) [inline packages/article/__tests__/related-articles-shared.js]: Is using import ... from ... not the preferred way of doing this?

File: packages/slice/__tests__/shared.js [hunk 3]
```diff
@@ -5,10 +5,10 @@ import context from "jest-context";
 
 import Slice from "../slice";
 
-const singleRelatedArticleFixture = require("../fixtures/single-related-article.json");
-const singleRelatedArticleNoImageFixture = require("../fixtures/single-related-article-no-image.json");
-const singleRelatedArticleNoLabelFixture = require("../fixtures/single-related-article-no-label.json");
-const singleRelatedArticleNoBylineFixture = require("../fixtures/single-related-article-no-byline.json");
+const singleRelatedArticleFixture = require("@times-components/article/related-articles/fixtures/single-related-article.json");
```
Comment (jimhunty) [inline packages/slice/__tests__/shared.js]: Shouldn't require fixtures from other packages. Doesnt work well with versioning.

Comment (colincclark) [reply packages/slice/__tests__/shared.js]: This is temporary - and only in the feature branch, see decription

File: packages/article/__tests__/related-articles-shared.js (hunk 2 above)
Comment (colincclark) [reply packages/article/__tests__/related-articles-shared.js]: this is the convention in the project

Comment (colincclark) [reply packages/article/__tests__/related-articles-shared.js]: see above - this is the convention

File: packages/slice/slice.stories.js [hunk 4]
```diff
@@ -1,46 +0,0 @@
-import React from "react";
```
[the remark this answers is not in the document]
Comment (colincclark) [reply packages/slice/slice.stories.js]: See description - stories to be jigged around on a later PR

File: packages/slice/package.json [hunk 5]
```diff
@@ -48,6 +48,7 @@
     "react-test-renderer": "16.2.0"
   },
   "dependencies": {
+    "@times-components/article": "0.31.3",
```
[the remark this answers is not in the document]
Comment (aronshamash) [reply packages/slice/package.json]: for the fixtures

File: packages/article/__tests__/related-articles-shared.js (hunk 2 above)
Comment (craigbilner) [reply packages/article/__tests__/related-articles-shared.js]: this was before we changed the babelrc

Comment (colincclark) [reply packages/article/__tests__/related-articles-shared.js]: amended in article and slice, fixtures and tests

**Pull request merged**